cmd/mqtt2prometheus.go

```diff
@@ -49,7 +49,7 @@ func main() {
 	mqttClientOptions.SetUsername(cfg.MQTT.User)
 	mqttClientOptions.SetPassword(cfg.MQTT.Password)
 
-	collector := metrics.NewCollector(2*time.Minute, cfg.Metrics)
+	collector := metrics.NewCollector(cfg.Cache.Timeout*time.Minute, cfg.Metrics)
```

Comment:
Just set it to cfg.Cache.Timeout. This allows the user to set arbitrary durations like 1m20s in the config. Your implementation requires the user to set multitudes of a minute